advanced_functionality/r_forecast_bring_your_own/R Forecast Bring-Your-Own Container Demo.ipynb

```diff
@@ -0,0 +1,333 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# R Forecastig Bring-Your-Own Container Demo\n",
+    "\n",
+    "This notebook demonstrates how to use the container to create an endpoint serving\n",
+    "forecasts made by the [R forecast package](https://cran.r-project.org/web/packages/forecast/index.html) using Amazon SageMaker."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import time\n",
+    "import urllib\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "import sagemaker"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# set this to the ECR image url of the container, returned by build_and_push.sh\n",
+    "# e.g. 123456789.dkr.ecr.us-east-1.amazonaws.com/r_forecast_bring_your_own:latest\n",
+    "CONTAINER_IMAGE = \"to_be_set\" \n",
```

Comment:
let's not rely on the customer figuring out to run build_and_push.sh, can we modify this cell so that we populate it on their behalf. We can either run build_and_push.sh in the notebook or use the default name.